.pre-commit-config.yaml

```
@@ -1,5 +1,5 @@
 ci:
-  autofix_prs: true
+  autofix_prs: false
```

I personally agree with this change, but let's do it in a separate PR so that others can review as well.

```suggestion
  autofix_prs: true
```